### ISSUE TITLE
NTR: Ureidopropionic acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8851]
New term request Ureidopropionic acid high in urine (elthat_111121051739):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000534
- label: Elevated urinary ureidopropionic acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: PMID:11675655
- def: The amount of ureidopropionic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8851
- synonyms: Ureidopropionic acid high in urine(NONE)
